.gitignore

```diff
@@ -1,3 +1,6 @@
+### IMPORTANT ###
+# When adding new filters please also consider adding them to `.dockerignore` if desired
```

Comment:
We should change this to be a warning to keep _**all**_ ignore files in sync. (I _think_ there might be an existing warning in some README or something somewhere if you could search for it, would definitely be better within the file though)

Reply:
I haven't noticed we have those section in other ignore files already and just the _most important_ ignore file does not have this information. I added a commit to unify them all.